libs/application/templates/legal-gazette/src/components/advert-display/AdvertDisplay.tsx

```
@@ -0,0 +1,132 @@
+          <Stack space={1}>
+            {title && (
+              <Text variant="medium" fontWeight="semiBold">
+                {title}
+              </Text>
+            )}
+            {html && (
+              <Box
+                position="relative"
+                overflow="auto"
+                className={s.bodyText}
+                dangerouslySetInnerHTML={{ __html: html }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)